### DISCUSSION TITLE
520的更新设备状态显示为空

### DISCUSSION THREAD

**Discussion by he110777** [Boos4721/openwrt#139] [Bugs/问题反馈]
如题。

**Comment by Boos4721** [comment]
ADD `else` will be ok
https://github.com/Boos4721/openwrt/commit/c3a1ffcce749500aef797304c9f1c20983b1c8de#diff-d693bb3b4961b09595bcfc61d9e8ec79c19a2ceff670200e0fd5593625de0be2L20
